### PR TITLE
chore(react-tree): stable release

### LINKED ISSUE #25266: Tree
## Preparation:

- [x] Open GitHub issues related to component
  - >  No issues were found
- [x] react-tree package scaffolded with the right ownership in CODEOWNERS
  - https://github.com/microsoft/fluentui/pull/25267
- [ ] Component Spec authored and reviewed
  - [link to spec in component package / PR]

## Implementation

[link to react-* package folder]

- [x] Component implementation
  - https://github.com/microsoft/fluentui/pull/25781
  - https://github.com/microsoft/fluentui/pull/26113
  - https://github.com/microsoft/fluentui/pull/26292
  - https://github.com/microsoft/fluentui/pull/26302
  - https://github.com/microsoft/fluentui/pull/26446
  - https://github.com/microsoft/fluentui/pull/26446
  - https://github.com/microsoft/fluentui/pull/26495
  - https://github.com/microsoft/fluentui/pull/26550
  - https://github.com/microsoft/fluentui/pull/26856
  - https://github.com/microsoft/fluentui/pull/26978
  - https://github.com/microsoft/fluentui/pull/26999
  - https://github.com/microsoft/fluentui/pull/27047
  - https://github.com/microsoft/fluentui/pull/27126
- [x] Storybook stories
  - https://github.com/microsoft/fluentui/pull/27270
- [ ] Add tests: Conformance, Unit, and VR -- add PRs to all
  - [x] Bundle size fixtures
  - [x] Conformance tests
  - [x] Unit tests
  - [x] E2E tests https://github.com/microsoft/fluentui/pull/27266, https://github.com/microsoft/fluentui/pull/27318
  - [x] VR tests
  - [x] Accessibility behavior tests
  - [x] Create an issue and run [manual accessibility tests](https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist): https://github.com/microsoft/fluentui/issues/28785
  - [x] Performance test scenario
- [ ] README.md covering basic usage
- [ ] MIGRATION.md guide (include v8 and v0)

## Validation

- [ ] Add and validate in UI Builder
- [ ] Add and validate in docs site
- [ ] Validate with token pipeline
- [ ] Validate in product
- [ ] Finalize migration guide
  - [ ] Author codemods

## Partner Deps / Asks

- PowerApps [[ref](https://powerbi.visualstudio.com/Trident/_workitems/edit/902480)]
- Other partner [data warehouse team] and [lakehouse team]

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. moves all code from `react-tree-preview` to `react-tree`
2. manually updates `react-tree-preview` everywhere to `react-tree` for the release
3. adds `react-tree` to `react-components` exports

## Related Issue(s)

- Fixes #25266

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                        |        Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------------------------------ | --------------------------: |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr> |  `4.162 kB`<br />`1.735 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>   |          `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>             | `34.611 kB`<br />`7.295 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>           | `19.282 kB`<br />`5.486 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                  |        `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6d3e683056ba52fc1df923d1c23d24abaa840fba'>6d3e683056ba52fc1df923d1c23d24abaa840fba</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">87</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">83</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Avatar-mount_base.html">661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Button-mount_base.html">328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Button-mount.html">323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Field-mount_base.html">1142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Field-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProvider-mount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProvider-mount.html">688</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-mount_base.html">87</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-mount.html">101</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">82</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">87</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">83</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/InfoButton-mount_base.html">10</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/InfoButton-mount.html">16</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/MakeStyles-mount_base.html">868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/MakeStyles-mount.html">880</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Persona-mount_base.html">1786</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/Persona-mount.html">1702</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/SpinButton-mount_base.html">1439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28845/perf-test-react-components/SpinButton-mount.html">1403</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by bsunderhus** [comment]
Blocked by https://github.com/microsoft/fluentui/pull/28848

File: change/@fluentui-react-components-0688cca2-da81-4fc4-b6a4-5eb65191cf56.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "chore: stable release 9.0.0 version of react-tree",
```
Comment (miroslavstastny) [inline change/@fluentui-react-components-0688cca2-da81-4fc4-b6a4-5eb65191cf56.json:3]: should be `feat`?

File: packages/react-components/react-tree/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-tree",
-  "version": "9.0.0-beta.32",
-  "description": "React components for building web experiences",
+  "version": "9.0.0",
+  "description": "New fluentui react package",
```
Comment (miroslavstastny) [inline packages/react-components/react-tree/package.json:4]: Fix the description

File: packages/react-components/react-tree/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-tree",
-  "version": "9.0.0-beta.32",
-  "description": "React components for building web experiences",
+  "version": "9.0.0",
```
Comment (miroslavstastny) [inline packages/react-components/react-tree/package.json:3]: Curious what the first published version will be :-)

**Review by miroslavstastny — APPROVED** [review]
179 files changed to just release a package as stable is really hard to review. I understand this is caused by a change in the unstable release process in the middle of the component development, hopefully this will not be the case going forward (cc @Hotell, @levithomason)

File: packages/react-components/react-tree/package.json (hunk 3 above)
Comment (bsunderhus) [reply packages/react-components/react-tree/package.json:3]: Yeah, I'm not sure if I should just keep the `9.0.0-beta.32` and do a minor or this approach of pinning it to `9.0.0` and doing no update version with the changefile

cc:  @Hotell ,  @tudorpopams , @marcosmoura

**Comment by bsunderhus** [comment]
> 179 files changed to just release a package as stable is really hard to review. I understand this is caused by a change in the unstable release process in the middle of the component development, hopefully this will not be the case going forward (cc @Hotell, @levithomason)

As the development was ongoing in the middle of the new [release process for v9 packages](https://github.com/microsoft/fluentui/wiki/new-release-process---v9-packages) I ended up kind of ignoring it for the sake of the development 🙈, now I'm paying the price with an overly complicated release 😅.

I believe for new packages this won't be this dramatic @miroslavstastny

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -239,7 +239,6 @@ packages/react-components/react-jsx-runtime @microsoft/teams-prg
 packages/react-components/react-toast @microsoft/teams-prg
```
Comment (fabricteam) [inline .github/CODEOWNERS:239]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28845&commitId=bb0a5c27e0ea71eab2b6747b5872c5ab09fc4c7f&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-components/react-tree/package.json (hunk 3 above)
Comment (bsunderhus) [reply packages/react-components/react-tree/package.json]: Just updated it to do a patch instead 😁

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**